bukuserver/server.py

```diff
@@ -117,6 +122,7 @@ def shell_context():
         return {'app': app, 'bukudb': bukudb}
 
     app.jinja_env.filters.update(util.JINJA_FILTERS)
+    app.jinja_env.globals.update(_p=_p)
```

Comment:
Exposing `_p()` (`pgettext()` aka translation with context) as a function in the templates